crates/common_enums/src/enums.rs

```diff
@@ -279,64 +279,67 @@ pub enum Country {
     SolomonIslands,
     Somalia,
     SouthAfrica,
-    SouthGeorgiaandtheSouthSandwichIslands,
+    SouthGeorgiaAndTheSouthSandwichIslands,
     SouthSudan,
     Spain,
     SriLanka,
     Sudan,
     Suriname,
-    SvalbardandJanMayen,
+    SvalbardAndJanMayen,
     Swaziland,
     Sweden,
     Switzerland,
     SyrianArabRepublic,
-    TaiwanProvinceofChina,
+    TaiwanProvinceOfChina,
     Tajikistan,
-    TanzaniaUnitedRepublicof,
+    TanzaniaUnitedRepublic,
     Thailand,
     TimorLeste,
     Togo,
     Tokelau,
     Tonga,
-    TrinidadandTobago,
+    TrinidadAndTobago,
     Tunisia,
     Turkey,
     Turkmenistan,
-    TurksandCaicosIslands,
+    TurksAndCaicosIslands,
     Tuvalu,
     Uganda,
     Ukraine,
     UnitedArabEmirates,
-    UnitedKingdomofGreatBritainandNorthernIreland,
-    UnitedStatesofAmerica,
+    UnitedKingdomOfGreatBritainAndNorthernIreland,
+    UnitedStatesOfAmerica,
     UnitedStatesMinorOutlyingIslands,
     Uruguay,
     Uzbekistan,
     Vanuatu,
-    VenezuelaBolivarianRepublicof,
-    VietNam,
+    VenezuelaBolivarianRepublic,
+    Vietnam,
     VirginIslandsBritish,
     VirginIslandsUS,
-    WallisandFutuna,
+    WallisAndFutuna,
     WesternSahara,
     Yemen,
     Zambia,
     Zimbabwe,
 }
 
+#[derive(Debug)]
+pub struct NumericCountryCodeParseError;
```

Comment:
Please implement `Display` for `NumericCountryCodeParseError` so that a useful error message is displayed whenever this error is logged.